apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -349,16 +372,30 @@ describe('files-storage controller (API)', () => {
 		});
 
 		describe(`with valid request data`, () => {
+			const setup = async () => {
+				const { result: uploadedFile } = await api.postUploadFile(`/file/upload/${validId}/schools/${validId}`);
```

Comment:
Couldn't you just create a `new FileRecordResponse(...)`? Not sure if it is better but I just wondered when I saw the async setup.

Reply:
I dont really understand what you suggest here. Creating a FileRecordResponse for what exactly?

Reply:
Nevermind... Was a misunderstanding from my side.